### DISCUSSION TITLE
dump file content

### DISCUSSION THREAD

**Discussion by cruiser2** [arendst/Tasmota#19307] [Q&A] [answered]
Hi,
decoding a dmp-file of a tasmotized Sonoff POW R3 20D I find in the WIFI section some entries like

  RgxAddress 192.168.99.1
  RgxNAPT 0
  RgxPassword "
  RgxSSId "
  RgxState 0
  RgxSubnet 255.255.255.0

I do not find any Tasmota command referring to those entries. 
Has anybody an idea about those entries?

**Comment by arendst** [comment, marked as answer]
Chk RangeExtender commands
